modules/microshift-adding-service-to-blueprint.adoc

```diff
@@ -48,7 +48,7 @@ EOF
 +
 [NOTE]
 ====
-The wildcard `*` in the commands uses the latest {microshift-short} RPMs. If you need a specific version, substitute the wildcard for the version you want. For example, insert `4.16.0` to download the {microshift-short} 4.16.0 RPMs.
+The wildcard `*` in the commands uses the latest {microshift-short} RPMs. If you need a specific version, substitute the wildcard for the version you want. For example, insert `4.17.0` to download the {microshift-short} 4.17.0 RPMs.
```

Comment:
```suggestion
The wildcard `*` in the commands uses the latest {microshift-short} RPMs. If you need a specific version, substitute the wildcard for the version you want. For example, insert `4.17.1` to download the {microshift-short} 4.17.1 RPMs.
```
There is no 4.17.0 release for MicroShift. It is coming out 4.17.1. We have to rebase on the released version of LVMS, so MicroShift is planned as a .1 z-stream release. cc @kowen-rh